docs/workflows-config-header.md

```diff
@@ -0,0 +1,41 @@
+This page provides documentation for `buildbuddy.yaml`, which can be placed
+at the root of your git repo to configure BuildBuddy [workflow](workflows) execution.
+
+_NOTE: This file only takes effect after you
+[enable workflows for the repo](workflows#enable-workflows-for-a-repo)._
+
+## Default / example config
+
+The following config is roughly equivalent to the one we use if you don't
+have a `buildbuddy.yaml` in your repo.
```

Comment:
We can't show the exact config since some data is populated dynamically, so I added the complete list of differences to this example config and removed the "roughly" language.